xlive/H2MOD.cpp

```diff
@@ -1912,16 +1881,20 @@ VOID CALLBACK UpdateDiscordStateTimer(HWND hwnd, UINT uMsg, UINT_PTR idEvent, DW
 
 void H2MOD::Initialize()
 {
-
-
-	
-	//HANDLE hThread = CreateThread(NULL, 0, Thread1, NULL, 0, NULL);
-
 	this->Base = (DWORD)game_info.base;
 
 	if (game_info.process_type == H2Types::H2Server)
 	{
 		this->Server = TRUE;
+
+		// Initialize teamspeak server
+		if (TS_server == NULL) 
```

Comment:
last I checked this happens when we apply hooks for the dedi? has that changed now?